src/lib/logger.ts

```diff
@@ -65,4 +65,16 @@ export function redactedPatch(patch: Record<string, Operation> = {}): Record<str
   return redactedCache;
 }
 
+export const logMutateErrorMessage = (e: Error): string => {
```

Comment:
Internally, we've noted that logging needs a closer look (see #1360). So, to facilitate that process when we get there, I moved a log-related function to logger.

Reply:
We'll actually keep this in `mutate-processor.ts` until we address #1360